docs/source/install/guide.rst

```diff
@@ -0,0 +1,225 @@
+
+    CentOS/RHEL
+
+  .. code-block:: bash
+
+    TBD
+
+Install StackStorm components
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  .. rst-class:: ubuntu
+
+    Ubuntu/Debian
+
+  .. code-block:: bash
+
+      sudo apt-get update
+      sudo apt-get install st2 st2-mistral
+
+  .. rst-class:: centos
+
+    CentOS/RHEL
+
+  .. code-block:: bash
+
+    TBD
+
+If you are not running RabbitMQ, MongoDB or PostgreSQL on the same box, or changed defauls,
+please adjust the settings:
+
+    * RabbitMQ connection at ``/etc/st2/st2.conf`` and ``/etc/mistral/mistral.conf``
+    * MongoDB at ``/etc/st2/st2.conf``
+    * PostgreSQL at ``/etc/mistral/mistral.conf``
+
+Setup Mistral Database
+~~~~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+    # Create Mistral DB in postgres
+    cat << EHD | sudo -u postgres psql
+    CREATE ROLE mistral WITH CREATEDB LOGIN ENCRYPTED PASSWORD 'StackStorm';
+    CREATE DATABASE mistral OWNER mistral;
+    EHD
+
+    #DZ: Do we REALLY need this, unless we upgrade Mistral?
+    /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+    #DZ: Do we REALLY need this? we don't use any default mistral actions and workflows...
+    /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+
+Configure SSH and SUDO
+~~~~~~~~~~~~~~~~~~~~~~
+To run local and remote shell actions, StackStorm uses a special system user (default ``stanley``).
+For remote linux actions, SSH is used. It is advised to configure identity file based SSH access on all remote hosts. We also recommend configuring SSH access to localhost for running examples and testing.
+
+* Take these steps on all boxes where you run stackstorm remote actions, **including** ``localhost``.
+
+  .. code-block:: bash
+
+    # Create an SSH system user
+    useradd stanley
+    mkdir -p /home/stanley/.ssh
+    chmod 0700 /home/stanley/.ssh
+
+    # Generate ssh keys on StackStorm box and copy over public key into remote box.
+    ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+    cp ${KEY_LOCATION}/stanley_rsa.pub /home/stanley/.ssh/stanley_rsa.pub
+
+    # Authorize key-base acces
+    cat /home/stanley/.ssh/stanley_rsa.pub >> /home/stanley/.ssh/authorized_keys
+    chmod 0600 /home/stanley/.ssh/authorized_keys
+    chown -R stanley:stanley /home/stanley
+
+    # Enable passwordless sudo
+    echo "stanley    ALL=(ALL)       NOPASSWD: SETENV: ALL" >> /etc/sudoers.d/st2
+
+* Enable passwordless sudo on for system user on StackStorm host
+  (required for local script actions, using ``local-shell-cmd`` and ``local-shell-script`` runners).
+
+  .. code-block:: bash
+
+    echo "stanley    ALL=(ALL)       NOPASSWD: SETENV: ALL" >> /etc/sudoers.d/st2
+
+* Adjust configuration in ``/etc/st2/st2.conf``:
+
+  .. sourcecode:: ini
+
+    [system_user]
+    user = stanley
+    ssh_key_file = /home/stanley/.ssh/stanley_rsa
+
+Check it all works
+~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+    st2 --version
```

Comment:
By the time we are here, we are root if I followed the guide strictly. Let's maybe fix the SSH keypair commands to use sudo on every line instead of root or add a line at end to logout as root.